server/resource_server/controller/controller_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"globe-and-citizen/layer8/server/resource_server/dto"
 	"globe-and-citizen/layer8/server/resource_server/models"
 	"globe-and-citizen/layer8/server/resource_server/utils"
+	testutils "globe-and-citizen/layer8/server/utils"
```

Comment:
Any reason for naming it `testutils`? Can we go with l8_utils or something relevant?

Reply:
Agree! Renamed to ```l8utils```